app/queries/get_duplicate_candidate_matches.rb

```diff
@@ -0,0 +1,25 @@
+class GetDuplicateCandidateMatches < ApplicationRecord
+  def self.call
+    ActiveRecord::Base.connection.exec_query(
+      "SELECT a.candidate_id, a.first_name, a.last_name, a.postcode, a.date_of_birth, email_address
+    FROM application_forms a
+    JOIN(
+      SELECT application_forms.last_name, application_forms.date_of_birth, application_forms.postcode
+      FROM application_forms
+      WHERE application_forms.previous_application_form_id IS NULL
+      AND application_forms.submitted_at IS NOT NULL
+      GROUP BY application_forms.last_name, application_forms.date_of_birth, application_forms.postcode
+      HAVING (count(*) > 1)
+    ) b
+    ON a.postcode = b.postcode
```

Comment:
@frankieroberto makes good points about whitespace and casing — maybe `TRIM()` and `LOWER()` these?